src/constants.js

```diff
@@ -337,6 +368,15 @@ const northDakotaTheme = {
       WHITE: ndColors.brandTealTint5,
       OTHER: ndColors.brandTealTint4,
     },
+    sentenceLengths: {
+      [SENTENCE_LENGTH_KEYS.lessThanOne]: ndColors.brandTeal,
+      [SENTENCE_LENGTH_KEYS.oneTwo]: ndColors.brandTealTint9,
+      [SENTENCE_LENGTH_KEYS.twoThree]: ndColors.brandTealTint7,
```

Comment:
Did you skip `brandTealTint8` (and `darkgreen8` above) on purpose?

Reply:
yeah that's how it is in the mocks